src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -228,6 +228,8 @@ export default function ConsumerIncentivesHomeScreen() {
         buttonPressed: RewardsScreenCta.ClaimRewards,
       })
     } else if (userIsVerified) {
+      // TODO: use navigate instead of replace currently navigate buggy on Android
+      // During initial load if account is restored from backup
```

Comment:
Should this be fixed before merging the PR?

Reply:
Removed comment in 103ad0f06c7e2bda92d3ae94334750720e6cffe2 and fixed in c61d4aaa96cc6e34e5d6f46347fe58fb20bd4fcb.